.readthedocs.yml

```diff
@@ -0,0 +1,24 @@
+# Configuration on how ReadTheDocs (RTD) builds our documentation
+# ref: https://docs.readthedocs.io/en/stable/config-file/v2.html
+
+# Required (RTD configuration version)
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: doc/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+formats: []
```

Comment:
This is a key part of the PR, to exclude HTMLZIP by default for example which causes failures.